CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_iterable_source v0.10.0
+[PR #39](https://github.com/fivetran/dbt_iterable_source/pull/39) contains the following updates:
+
+## Bug Fix 
+- Removes the source freshness tests in the `src_iterable.yml`. This was originally causing run errors for users where tables weren't present in their schema but listed in the `src_iterable.yml`.
+
+## Breaking Change
+- Updates the materialization in `stg_iterable__user_history` from table to view in order to improve performance. We encourage you to run a `--full-refresh` to ensure there are no issues.
```

Comment:
Small request but we should list the breaking change about the bugfix.

Reply:
yes! added